resources/js/tall-stack/dynamic/reorderSiteAwards/handleResetOrder.ts

```diff
@@ -0,0 +1,52 @@
+import { collectMappedTableRows } from './collectMappedTableRows';
+import type { MappedTableRow } from './index';
+import { computeDisplayOrderValues } from './index';
+
+declare global {
+  interface Window {
+    postAllAwardsDisplayOrder: (awards: Partial<MappedTableRow>[]) => void;
+  }
+}
+
+export function handleResetOrder(): void {
+  if (!confirm('This will reset the order of all awards. Are you sure?')) {
+    return;
+  }
+
+  const rows = Array.from(document.querySelectorAll<HTMLTableRowElement>('.award-table-row'));
+
+  // Sort rows by date of aquisition (ascending)
+  sortAwardsByAwardDate(rows);
+
+  for (const row of rows) {
+    const awardKind = row.getAttribute('data-award-kind');
+    document
+      .querySelector<HTMLTableElement>(`#${awardKind}-reorder-table`)
+      ?.querySelector('tbody')
+      ?.appendChild(row);
+  }
+
+  const mappedTableRows = collectMappedTableRows();
+
+  window.postAllAwardsDisplayOrder(computeDisplayOrderValues(mappedTableRows));
```

Comment:
UX: I recommend we remove this line.

The page has a "Save All Changes" button which users expect to use for saving. Having this button auto-save while other operations don't might be considered a dishonest design.

After seeing their awards resorted, users might decide they don't like what they see. By requiring the click to Save All Changes, users can discard the change if they decide they don't want to go through with it.